deployments-npm-package/__tests__/test_package.ts

```diff
@@ -0,0 +1,113 @@
+import path from 'path';
+import { execSync } from 'child_process';
+import fs from 'fs/promises';
+
+const pathToArtifacts = `${__dirname}/../../deployments/artifacts`;
+/**
+ * We read the artifacts directory and get read all the json artifacts, from there we extract the contracts
+ * inside of the base or chain key and return them as an array.
+ *
+ * @returns {Promise<{ [key: string]: string[] }>}
+ */
+const getContracts = async () => {
+  const contractsPerNetwork: { [key: string]: string[] } = {};
+  try {
+    const artifactItems = await fs.readdir(pathToArtifacts);
+
+    for (const item of artifactItems) {
+      const artifactPath = path.join(pathToArtifacts, item);
```

Comment:
If you apply the above change, you would have to refactor the other places where `artifactsItems` occurs. 
We could have something like:
`for (const network of networks) { ... }`

Once you do these changes, you will see things will start making more sense in this code block :D